README.md

```diff
@@ -41,6 +41,33 @@ Admin Editing Users:
 
 ![edit user](readme_media/edituser.gif "edituser")
 
+## Gettin up and running with Docker and docker-compose:
```

Comment:
Can you push this section down to below the regular setup (i.e. should show up after "Running the app" section).

Reply:
Done!